cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -25,6 +25,8 @@ import           Control.Monad.Trans.Except.Extra (firstExceptT, handleIOExceptT
 import qualified Cardano.Binary as CBOR
 
 import qualified Shelley.Spec.Ledger.PParams as Shelley
+import           Shelley.Spec.Ledger.Scripts ()
+import           Cardano.Ledger.ShelleyMA.TxBody ()
```

Comment:
I suppose that in the future the instances imported here will already be imported by other imports, e.g., `Cardano.Api.Typed`.